File: leakcanary-support-fragment/src/androidTest/java/com/squareup/leakcanary/tests/InstrumentationTestApplication.java
package com.squareup.leakcanary.tests;

import android.app.Application;
import com.squareup.leakcanary.InstrumentationLeakDetector;

public class InstrumentationTestApplication extends Application {
  @Override public void onCreate() {
    super.onCreate();
    InstrumentationLeakDetector.Companion.instrumentationRefWatcher(this)
        .buildAndInstall();
  }
}

File: leakcanary-support-fragment/src/androidTest/java/com/squareup/leakcanary/tests/InstrumentationTestApplication.kt
package com.squareup.leakcanary.tests

import android.app.Application
import com.squareup.leakcanary.InstrumentationLeakDetector

class InstrumentationTestApplication : Application() {
  override fun onCreate() {
    super.onCreate()
    InstrumentationLeakDetector.updateConfig()
  }
}
